client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/FamilyContactsDialog/FamilyContactsTable/FamilyContactsTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody } from '@material-ui/core';
+
+import InvolvedContact from 'models/InvolvedContact';
+import InteractedContactFields from 'models/enums/InteractedContact';
+
+import useStyles from './FamilyContactsTableStyles';
+import FamilyContactsTableHeadersNames, { FamilyContactsTableHeaders } from './FamilyContactsTableHeaders';
+
+const birthDateFormat = 'dd/MM/yyyy';
+
+const FamilyContactsTable: React.FC<Props> = (props: Props) => {
+
+    const { familyMembers, className } = props;
+
+    const classes = useStyles();
+
+    const convertToIndexedRow = (row: InvolvedContact) : IndexedContactRow => ({
+        [FamilyContactsTableHeadersNames.FAMILY_RELATIONSHIP]: row.familyRelationship,
+        [FamilyContactsTableHeadersNames.FIRST_NAME]: row.firstName,
+        [FamilyContactsTableHeadersNames.LAST_NAME]: row.lastName,
+        [FamilyContactsTableHeadersNames.IDENTIFICATION_TYPE]: row.identificationType,
+        [FamilyContactsTableHeadersNames.IDENTIFICATION_NUMBER]: row.identificationNumber,
+        [FamilyContactsTableHeadersNames.BIRTH_DATE]: row.birthDate,
+        [FamilyContactsTableHeadersNames.PHONE_NUMBER]: row.phoneNumber,
+        [FamilyContactsTableHeadersNames.ADDITIONAL_PHONE_NUMBER]: row.additionalPhoneNumber,
+        [FamilyContactsTableHeadersNames.ISOLATION_CITY]: row.isolationCity
+    })
+
+    const getTableCell = (row: IndexedContactRow, cellName: string) => {
+        const cellValue = row[cellName as FamilyContactsTableHeadersNames];
+        if (!cellValue) return '---'; 
+        switch (cellName) {
+            case FamilyContactsTableHeadersNames.BIRTH_DATE: {
+                return format(new Date(cellValue), birthDateFormat);
+            }
+            default: 
+                return cellValue;
+        }
+    }
+
+    return (
+        <TableContainer className={className} component={Paper}>
+            <Table stickyHeader>
+                <TableHead>
+                    <TableRow>
+                        {
+                            Object.keys(FamilyContactsTableHeaders).map(cellName => {
```

Comment:
why not mapping Object.values(FamilyContactsTableHeaders)?

Reply:
Because that would be the hebrew name and it wouldnt work...

Reply:
Do you mean because the order of the headers will be different?

Reply:
Check the FamilyContactsTableHeaders, and see the values are headers hebrew display

Reply:
Yeah but you display the Hebrew value anyway in the return statement. We are trying to understand what is the difference between mapping the header values directly instead of mapping the keys and then accessing the same values.